hooks/base16-jqp.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# ----------------------------------------------------------------------
+# Setup config variables and env
+# ----------------------------------------------------------------------
+#
+
+BASE16_JQP_CONFIG_FILE="${BASE16_JQP_CONFIG_FILE:-$BASE16_CONFIG_PATH/jqp_theme.yaml}"
+
+if [ ! -e "$BASE16_JQP_CONFIG_FILE" ]; then
+  touch "$BASE16_JQP_CONFIG_FILE"
+fi
+
+read -r current_theme_name < "$BASE16_SHELL_THEME_NAME_PATH"
+
+# ----------------------------------------------------------------------
+# Execution
+# ----------------------------------------------------------------------
+#
+cat <<EOF >! $BASE16_JQP_CONFIG_FILE
```

Comment:
Use https://github.com/tinted-theming/tinted-jqp intead of generating the theme here.

Reply:
Done https://github.com/tinted-theming/tinted-shell/pull/57/commits/98744264c2deec87907b935746a80fb924631b64